### PR TITLE
assertions: improve diffs readability

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

see https://github.com/codeready-toolchain/toolchain-e2e/pull/400#issuecomment-953689054

### CODE REVIEW

**Review by MatousJobanek — APPROVED** [review]

**Review by ranakan19 — APPROVED** [review]
LGTM

**Comment by rajivnathan** [comment]
Nice 👍

**Comment by xcoulon** [comment]
> Nice 👍

thanks @rajivnathan and my apologies, I realize I forgot to add you as a reviewer :/ sorry about that :(

File: testsupport/wait/diff.go [hunk 1]
```diff
@@ -1,55 +1,9 @@
...
-			_, _ = buff.WriteString(strings.Replace(text, " ", "-", 1))
-			_, _ = buff.WriteString("\x1b[0m")
-		case diffmatchpatch.DiffEqual:
-			_, _ = buff.WriteString(text)
-		}
-	}
-
-	return buff.String()
+func Diff(expected, actual interface{}) string {
+	return cmp.Diff(expected, actual)
```
Comment (MatousJobanek) [inline testsupport/wait/diff.go:8]: so simple :star_struck: :+1:

**Review by MatousJobanek — APPROVED** [review]
:rocket:

**Pull request merged**